terra-jupyter-r/Dockerfile

```diff
@@ -121,7 +121,7 @@ RUN pip3 -V \
 
 RUN R -e 'install.packages("BiocManager")' \
     ## check version
-    && R -e 'BiocManager::install(version="3.14", ask=FALSE)' \
+    && R -e 'BiocManager::install(version="3.15", ask=FALSE)' \
```

Comment:
we're updating r version in this PR too?

Reply:
bioconductor released 3.15 which got pulled in automatically to `terra-jupyter-r` - this line just checks the version of bioconductor that gets installed

Reply:
why are we updating bioconductor in this PR?

Reply:
we aren't explicitly updating the version - without any changes to the r image, [this line ](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-r/Dockerfile#L124)fails because we're installing bioconductor version 3.15

alternatively, we could try to pin version 3.14 in [this line](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-r/Dockerfile#L122) to avoid bumping the bioconductor version

Reply:
ah yea...pinning that version is probably better... @nturaga has a seaparate PR for uprading bioconductor

Reply:
@nturaga's [PR](https://github.com/DataBiosphere/terra-docker/pull/318) only updates Bioconductor in the Anvil RStudio image and not in the terra-jupyter-r image

Reply:
he hasn't PRed that one yet...I paired with him the other day

Reply:
Yes, this is because the R-4.2.0 update on the apt repository isn't available yet. We need to wait to do this with `apt`. The other way to do this is to do a source installation of R, which would involve compiling everything from scratch.

Reply:
let's separate things...and pin the version in https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-r/Dockerfile#L122 for this one

Reply:
OH that makes SO much sense. I updated bioconductor for terra-jupyter-r and that was causing issues so I'll go ahead and try to pin version 3.14 until we're ready to bump R